<ui version="4.0" >
 <class>SlideshowClip_UI</class>
 <widget class="QDialog" name="SlideshowClip_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="KLineEdit" name="clip_name" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Frame Duration</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="KRestrictedLine" name="clip_duration" >
     <property name="inputMask" >
      <string>99:99:99:99; </string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="KUrlRequester" name="folder_url" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="KComboBox" name="image_type" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Image Type</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="KListWidget" name="icon_list" />
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="label_info" >
     <property name="text" >
      <string>No image found</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QCheckBox" name="slide_fade" >
     <property name="text" >
      <string>Crossfade</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QCheckBox" name="luma_fade" >
     <property name="text" >
      <string>Luma File</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2" >
    <widget class="KComboBox" name="luma_file" />
   </item>
   <item row="5" column="1" >
    <widget class="QCheckBox" name="slide_loop" >
     <property name="text" >
      <string>Loop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KRestrictedLine</class>
   <extends>KLineEdit</extends>
   <header>krestrictedline.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SlideshowClip_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SlideshowClip_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
